metarecord/views/function.py

```diff
@@ -255,11 +255,12 @@ def get_version_history(self, obj):
 class FunctionFilterSet(django_filters.FilterSet):
     class Meta:
         model = Function
-        fields = ('valid_at', 'version')
+        fields = ('valid_at', 'version', 'classification__code', 'information_system')
 
     valid_at = django_filters.DateFilter(method='filter_valid_at')
     modified_at__lt = django_filters.DateTimeFilter(field_name='modified_at', lookup_expr='lt')
     modified_at__gt = django_filters.DateTimeFilter(field_name='modified_at', lookup_expr='gt')
+    information_system = django_filters.CharFilter(field_name='information_system', method='filter_information_system')
```

Comment:
Not sure, but I think you might get away with not having the separate filtering method with something like `CharFilter(field_name='phases__actions__records__attributes__InformationSystem', lookup_expr='icontains')`